entry_types/scrolled/package/src/frontend/SoundDisclaimer.js

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import i18n from 'i18n-js';
```

Comment:
All other places import i18n with a capital first letter. Let's stick with that.